test/JDBC/input/storedProcedures/Test-sp_helpdbfixedrole-vu-verify.sql

```diff
@@ -0,0 +1,55 @@
+INSERT INTO test_sp_helpdbfixedrole_tbl (DbFixedRole, Description) EXEC sp_helpdbfixedrole
+GO
+
+SELECT DbFixedRole, Description FROM test_sp_helpdbfixedrole_tbl
+GO
+
+TRUNCATE TABLE test_sp_helpdbfixedrole_tbl
+GO
+
+INSERT INTO test_sp_helpdbfixedrole_tbl (DbFixedRole, Description) EXEC sp_helpdbfixedrole 'db_owner'
+GO
+
+SELECT DbFixedRole, Description FROM test_sp_helpdbfixedrole_tbl
+GO
+
+TRUNCATE TABLE test_sp_helpdbfixedrole_tbl
+GO
+
+INSERT INTO test_sp_helpdbfixedrole_tbl (DbFixedRole, Description) EXEC sp_helpdbfixedrole 'DB_OWNER    '
+GO
+
+SELECT DbFixedRole, Description FROM test_sp_helpdbfixedrole_tbl
+GO
+
+TRUNCATE TABLE test_sp_helpdbfixedrole_tbl
+GO
+
+INSERT INTO test_sp_helpdbfixedrole_tbl (DbFixedRole, Description) EXEC sp_helpdbfixedrole 'db_accessadmin'
+GO
+INSERT INTO test_sp_helpdbfixedrole_tbl (DbFixedRole, Description) EXEC sp_helpdbfixedrole 'DB_securityadmin'
+GO
+INSERT INTO test_sp_helpdbfixedrole_tbl (DbFixedRole, Description) EXEC sp_helpdbfixedrole 'db_ddladmin   '
+GO
+INSERT INTO test_sp_helpdbfixedrole_tbl (DbFixedRole, Description) EXEC sp_helpdbfixedrole 'DB_backupoperator     '
+GO
+INSERT INTO test_sp_helpdbfixedrole_tbl (DbFixedRole, Description) EXEC sp_helpdbfixedrole 'db_datareader'
+GO
+INSERT INTO test_sp_helpdbfixedrole_tbl (DbFixedRole, Description) EXEC sp_helpdbfixedrole 'db_datawriter'
+GO
+INSERT INTO test_sp_helpdbfixedrole_tbl (DbFixedRole, Description) EXEC sp_helpdbfixedrole 'db_denydatareader'
+GO
+INSERT INTO test_sp_helpdbfixedrole_tbl (DbFixedRole, Description) EXEC sp_helpdbfixedrole 'db_denydatawriter'
+GO
+
+SELECT DbFixedRole, Description FROM test_sp_helpdbfixedrole_tbl
+GO
+
+EXEC sp_helpdbfixedrole ''
+GO
+
+EXEC sp_helpdbfixedrole '   Db_owner    '
+GO
+
+EXEC sp_helpdbfixedrole 'error'
+GO
```

Comment:
Since this is a new test file, you'll need to add it to upgrade test schedule. In `upgrade` folder, for each version (13_4 to latest), please add Test-sp_helpdbfixedrole to the schedule.

Another thing is that, we need another upgrade test Test-sp_helpdbfixedrole-dep, you can refer to `input/storedProcedures/Test-sp_helpsrvrolemember-dep`. The dep test is to test object dependency for releases after this cycle. 
In this dep test, we need to prepare one procedure, one function and one view that depend on sp_helpdbfixedrole.
For now the dep test doesn't need to be added into any schedule. At the start of next cycle we'll add it to latest/schedule.